tests/_test_utils/torch_quantization/quantize_common.py

```diff
@@ -116,40 +119,160 @@ def save_restore_test(model_cls, device, quant_config, compress=False, version=N
         mto.restore_from_modelopt_state(model_ref, state_dict)
 
 
-def tensor_parallel_test_helper(model, config, tp_group, dp_group):
-    # The input to fist layer, the column parallel should be the same across all tp ranks
+def _reduce_quantizer_attr(quantizer, attr: str, op=dist.ReduceOp.MAX, group=None):
```

Comment:
nit: Can we rename this to _distributed_attr_check
```suggestion
def _distributed_attr_check(quantizer, attr: str, op=dist.ReduceOp.MAX, group=None):
```